zdm/survey.py

```diff
@@ -153,9 +154,9 @@ def process_survey_file(self,filename:str, NFRB:int=None,
             self.NFRB=keys.count('FRB')
         else:
             self.NFRB = NFRB
-        if self.NFRB==0:
-            raise ValueError('No FRBs found in file '+filename) #change this?
-
+        #if self.NFRB==0:
```

Comment:
Correct, we shouldn't be using it. Maybe we shortly shift this to a separate "old_survey.py" file, to make development easier? However, I had to change this, since it's vital to allow surveys to have no FRBs in them. So I'll keep this code there, but we can discuss what to do about old_survey. I think we need to keep the code for posterity, but maybe shift it somewhere? Or I guess, this is what version control is for. Once we are 100% sure we have converted over all old .dat files, then we delete